BSP/STM32F413/Src/BSP_I2C.c

```diff
@@ -2,6 +2,9 @@
 
 #include "BSP_I2C.h"
 #include "stm32f4xx.h"
+#include "stm32f4xx_gpio.h"
+#include "stm32f4xx_rcc.h"
+#include "stm32f4xx_i2c.h"
```

Comment:
Did all these files need to be changed? They worked before without the includes.

Reply:
I don't remember changing this file so I'll revert these changes